pandas/conftest.py

```diff
@@ -145,6 +145,10 @@ def pytest_collection_modifyitems(items, config) -> None:
             "(Series|DataFrame).bool is now deprecated and will be removed "
             "in future version of pandas",
         ),
+        (
+            "pandas.core.generic.NDFrame.clip",
```

Comment:
Should we add this one to the whatsnew as well?

Reply:
will update

Reply:
updated + green